src/apiProxy/api-proxy.js

```diff
@@ -27,20 +37,28 @@ import { duotoneRef } from '../util/duotone';
  * @param response {String} the raw response body text from an API call
  * @return responseObj the JSON-parsed text, possibly with error info
  */
-export const parseApiResponse = response => {
-	let responseObj;
-	let error;
+export const parseApiResponse = ([response, body]) => {
+	let value;
+	const flags = parseResponseFlags(response);
 
-	try {
-		responseObj = JSON.parse(response);
-	} catch(e) {
-		error = `API response was not JSON: "${response}"`;
+	if (response.statusCode > 299) {
```

Comment:
I think it's more accurate to say I'm handling `1xx` as 'not errors', but yeah that's not really an edge case I'm explicitly handling. However, I don't think it's actually possible to get a `1xx` code from an ajax call? I'm curious if you know if it's possible.